food.js

```diff
@@ -0,0 +1,76 @@
+// You are working as a junior developer team for Acme, Inc. This company sells
+// dog food. Each brand of dog food is represented by an object in an array you
+// get back from the (fictional database) in JSON format.
+
+// Your first task is to build a product page that displays all brands of dog
+// food, the different types, with the price and size for each container volume.
+
+// Once you have all of that information listed, you need to create a second
+// JSON file to represent the new product line for Acme, Inc. They now want to
+// sell cat food. Your product owner gives you a spreadsheet with all the new
+// data in it. It is your job to get this data represented correctly in JSON
+// format and then update your page to now show cat food, along with dog food.
+
+var container = document.getElementById("table-container");
+var catContainer = document.getElementById("cat-container");
+
+function pullInDogFood() {
+	var dogFoodData = JSON.parse(this.responseText);
+  dogTable(dogFoodData.dog_brands);
+}
+
+function pullInCatFood() {
+	var catFoodData = JSON.parse(this.responseText);
+	catTable(catFoodData.cat_brands);
+}
+
+function error() {
+	throw "Data failed to load. Try again";
```

Comment:
Always throw an error object: `throw new Error("Data failed to load. Try again");`